homeassistant/components/home_connect/api.py

```diff
@@ -324,6 +415,21 @@ class CoffeeMaker(DeviceWithPrograms):
         {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.WarmMilk"},
         {"name": "ConsumerProducts.CoffeeMaker.Program.Beverage.Ristretto"},
         {"name": "ConsumerProducts.CoffeeMaker.Program.CoffeeWorld.Cortado"},
+        {"name": "ConsumerProducts.CoffeeMaker.Program.CoffeeWorld.CafeCortado"},
```

Comment:
Please make a separate PR for these changes.

Reply:
Removed.